docs/content/SplitPageLayout.mdx

```diff
@@ -0,0 +1,313 @@
+---
+title: SplitPageLayout
+componentId: split_page_layout
+status: Draft
+# description: TODO
```

Comment:
@vdepizzol Do you have a concise description for SplitPageLayout that we can use here?

Reply:
@colebemis wdyt?

```suggestion
description: Provides structure for a split page layout, providing independent scrolling for the pane and content regions. Useful for responsive list/detail patterns, when an item in the pane updates the page content on selection.
```

Reply:
I like it 👍